src/apiData/utilities.js

```diff
@@ -0,0 +1,56 @@
+const cleanApiData = {
+  getAllMovies: (data) => {
+    return data.movies.map(movie => {
+      return {
+        id: movie.id,
+        poster: movie.poster_path
+      }
+    })
+  },
+
+  getAMovie: (data) => {
```

Comment:
The naming of this key is confusing. It seems to clean movie data, not get a movie like it states.